### PR TITLE
Use cases descriptions correspondence PR

### PR DESCRIPTION
**Pull request by Arcapas**
Hi team,

## Description
This PR aims to create a correspondence between the defined capabilities and those described in the Getting started - Use cases section:
- [x] 01. Research and define the state of correspondence. More information on issue #4101               
- [x] 02. Write a new page for the **Security Analytics** capability in the Getting started > Use cases.
- [x] 03. Write a new page for the **Intrusion Detection** capability in the Getting started > Use cases.
- [x] 04. Reutilize the Rootkits detection and System inventory section for the new **Intrusion Detection** page. 
- [ ] 05. Review the tone of the remaining eight use cases to keep the same tone in all the section.
     - [x] Log data analysis tone adjustment
     - [x] File integrity monitoring tone adjustment
     - [x] Vulnerability detection tone adjustment
     - [x] Configuration assessment tone adjustment
     - [x] Incident response tone adjustment
     - [x] Regulatory compliance tone adjustment
     - [x] Cloud security tone adjustment
     - [x] Containers security tone adjustment

## Checks
- [ ] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

Regards,
Damián

### CODE REVIEW

File: source/getting-started/use_cases/security_analytics.rst [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
+Its agent modular architecture allows Wazuh to collect security data from different components. This way, Wazuh generates alerts and events, fetching data from the endpoints where the agents are installed. 
+
+Below you can see a list of these components:
+
+- **File integrity monitoring (FIM)** reports when files are created, deleted, or modified in the monitored directories. 
+- **Malware detection** looks for hidden processes, files, and ports to detect anomalies and the possible presence of rootkits. 
+- **Command execution** can be used to achieve different purposes such as to monitor hard disk space left and get a list of last logged-in users. 
+- **Log collector** reads flat log files, collecting operating system and application log messages.
+- **System inventory** runs regular scans, collecting inventory data including network interfaces, operating system version, installed applications, running processes, and a list of open ports. 
+- **Security configuration assessment (SCA)** provides configuration assessment based on the Center of Internet Security (CIS) benchmarks.
```
[the remark this answers is not in the document]
Comment (Arcapas) [reply source/getting-started/use_cases/security_analytics.rst:23]: Solved.

File: source/getting-started/use_cases/security_analytics.rst [hunk 2]
```diff
@@ -1,8 +1,29 @@
...
+
+This way, security analytics tools help users perform thread prevention actions collecting log messages. Users can also get a list of last logged-in users and logs from files changed in the monitored file systems.
```
[the remark this answers is not in the document]
Comment (Arcapas) [reply source/getting-started/use_cases/security_analytics.rst:25]: Solved.

File: source/getting-started/use_cases/intrusion_detection.rst [hunk 3]
```diff
@@ -1,12 +1,44 @@
...
+  
+ 
 .. _intrusion_detection:
 
+
 Intrusion detection
 ===================
-Wazuh agents...
+
+Wazuh owns a highly effective group of capabilities to perform intrusion detection within user's systems. These set of capabilities include :ref:`File Integrity monitoring (FIM) <file_integrity>`, :ref:`Log data analysis <log_analysis>`, :ref:`Rootkits detection <rootkits_detection>`, and :ref:`System inventory <system_inventory>`. Which are useful to identify suspicious anomalies within files, processes, and ports.
```
[the remark this answers is not in the document]
Comment (Arcapas) [reply source/getting-started/use_cases/intrusion_detection.rst:13]: Solved

**Review by MarVergara — COMMENTED** [review]
LGTM after comments from editorial review are addressed.

**Review by MarVergara — APPROVED** [review]

File: source/getting-started/use_cases/intrusion_detection.rst [hunk 4]
```diff
@@ -1,12 +1,50 @@
...
+  
+ 
 .. _intrusion_detection:
 
+
 Intrusion detection
 ===================
-Wazuh agents...
+
+Wazuh owns a highly effective group of tools to perform intrusion detection within user's systems. This set of tools include :ref:`File Integrity monitoring (FIM) <file_integrity>`, :ref:`Log data analysis <log_analysis>`, :ref:`Rootkits detection <rootkits_detection>`, and :ref:`System inventory <system_inventory>` which are useful to identify suspicious anomalies within files, processes, and ports.
```
Comment (MarVergara) [inline source/getting-started/use_cases/intrusion_detection.rst:13]:
```suggestion
Wazuh owns a highly effective array of tools to perform intrusion detection within user's systems. This set of tools include :ref:`File Integrity monitoring (FIM) <file_integrity>`, :ref:`Log data analysis <log_analysis>`, :ref:`Rootkits detection <rootkits_detection>`, and :ref:`System inventory <system_inventory>` which are useful to identify suspicious anomalies within files, processes, and ports.
```

**Review by davidfmiranda — APPROVED** [review]

File: source/getting-started/use_cases/active_response.rst [hunk 5]
```diff
@@ -29,7 +29,7 @@ Below you can find the configuration used to define two scripts that are used fo
       <timeout_allowed>yes</timeout_allowed>
     </command>
 
-On top of the defined commands, active responses set the conditions that need to be met to trigger them. Below is an example of configuration that would trigger the ``firewall-drop`` command when the log analysis rule ``100100`` is matched.
+Active responses set the conditions that need to be met to trigger them on top of the defined commands. Below is an example of a configuration that would trigger the ``firewall-drop`` command when the log analysis rule ``100100`` is matched.
```
[the remark this answers is not in the document]
Comment (Arcapas) [reply source/getting-started/use_cases/active_response.rst:32]: Depending on the context, "Incident response" refers to the capability, and "active response" refers to the command.

File: source/getting-started/use_cases/security_analytics.rst [hunk 6]
```diff
@@ -1,8 +1,42 @@
...
+- **Log collector** reads flat log files, collecting operating system and application log messages.
+- **System inventory** runs regular scans, collecting inventory data including network interfaces, operating system version, installed applications, running processes, and a list of open ports. 
+- **Security configuration assessment (SCA)** provides a continuous evaluation based on the Center of Internet Security (CIS) benchmarks.
+
+Security analytics tools give users the needed visibility to perform thread prevention actions. The :ref:`FIM <file_integrity>` module is a good example because it detects changes that could compromise data and creates alerts. Then, the agent can be configured to trigger an Active Response in reply to the alerts.
+
+In addition, these tools assist users in detecting the possible presence of rootkits and collecting inventory data from different points such as operating systems, processes, applications, ports, and network interfaces. FIM results can be observed in :ref:`Kibana app<kibana_app>` in the *Overview > Integrity monitoring* section:
```
[the remark this answers is not in the document]
Comment (Arcapas) [reply source/getting-started/use_cases/security_analytics.rst:27]: I made a commit with this change.

**Pull request merged**